package.json

```diff
@@ -72,16 +75,20 @@
     "stylus": "^0.54.0",
     "stylus-loader": "^2.4.0",
     "svg-sprite-loader": "^0.0.31",
+    "wdio-mocha-framework": "^0.5.9",
     "webpack": "^1.14.0",
     "webpack-dev-server": "^1.16.0",
     "webpack-merge": "^1.0.2"
   },
   "dependencies": {
+    "babel-register": "^6.24.0",
+    "chai": "^3.5.0",
     "classnames": "^2.2.0",
     "cozy-ui": "^3.0.0-beta17",
     "date-fns": "^1.26.0",
     "hammerjs": "^2.0.8",
     "justified-layout": "^2.1.0",
+    "mocha": "^3.2.0",
```

Comment:
Concerning:
- babel-register
- chai
- mocha
- wdio-visual-regression-service
- webdriverio

I guess it worth to be added in *dev dependencies* instead of *dependencies*. Why did you make this choice? Maybe I miss some practical information.